lib/net/seeds/main.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
 
 module.exports = [
-  '193.70.122.58'
```

Comment:
Maybe there are additional seed nodes we could add just in case these go offline. I will look into what other public nodes I can connect to and we could add all of them. Last time I saw there were at least 15 public nodes fully portforwarded. I know Mediciland is running one still (fcoin specifically), let me do some research and I will get back to this!